examples/assets/scripts/utils/cubemap-renderer.js

```diff
@@ -0,0 +1,106 @@
+// How to use in the Editor:
+// - create entity with Camera component - position of the entity defines where the cubemap is rendered from
+//   and properties of the Camera are used to render cubemap (adjust near / far distance, clearing, layers and other properties)
+//   Note: the layers should contain all layers visible by cybemap camera.
+// - to use generated cube map, you can access it like this using script:
+//   material.cubeMap = entity.script.cubemapRenderer.cubeMap;
+
+var CubemapRenderer = pc.createScript('cubemapRenderer');
+
+CubemapRenderer.attributes.add('resolution', {
+    title: 'Resolution',
+    description: 'Resolution of one side of a cubemap. Use power of 2 resolution if you wish to use Mipmaps.',
+    type: 'number',
```

Comment:
I'd consider making this an enum with:
* 16x16
* 32x32
* 64x64
* 128x128
* 256x256
* 512x512
* 1024x1024
* 2048x2048

Reply:
It feels like this would limit the options for no good reason.

Reply:
Well, the good reason is that it prevents users from setting `100` or `8192`, say.

Reply:
An alternative is you could:
1. snap to the closest power of two
2. clamp between min and max cubemap resolutions supported by the graphics device

Reply:
setting 100 is perfectly valid though? And setting it to out of range number should be responsibility of the Texture to take care of and clamp to the platform limits I think, as this class simply creates a Texture.
I tried 100 and that works fine. I also tried 15000 and that ended up with out of video memory error :)

Reply:
Ahhh, my mistake. I thought cubemaps had to be power of two, even if not mipmapped. But yeah, I guess that's wrong.

But on a device that only supports, say, 512 max cubemap resolution, if you set it to 2048, the engine doesn't just handle that for you, does it?

Reply:
I added handling of maxTextureSize .. but perhaps we should handle this in Texture's constructor as well?

Reply:
Possibly. But resolving this for now.